thorlcr/master/mawatchdog.cpp

```diff
@@ -61,16 +61,18 @@ class CMachineStatus
 };
 
 
-CMasterWatchdogBase::CMasterWatchdogBase() : threaded("CMasterWatchdogBase")
+CMasterWatchdog::CMasterWatchdog(bool startNow) : threaded("CMasterWatchdogBase")
```

Comment:
minor: I'm not sure why the thread would be started in the constructor, as there wouldn't be nodes registered so early. 
 Any reason this is start should be supported during construction?

Reply:
slaves could register a split second after the ctor, so it would make some sense to start here.
But, and I don't remember the exact reasoning, it was decided to delay monitoring until everything was up though (probably relates to the original idea that slaves could come and go which never happened).

Not new, so will leave it as it is.